Makefile

```diff
@@ -40,29 +40,32 @@ help:
 
 install: derived
 	@install -d $(DESTFOLDER)
-	@install ${BUILD_DIR}/3270Narrow.ttf ${BUILD_DIR}/3270Medium.ttf ${BUILD_DIR}/3270SemiNarrow.ttf $(DESTFOLDER)
+	@install ${BUILD_DIR}/3270Condensed-Regular.ttf ${BUILD_DIR}/3270-Regular.ttf ${BUILD_DIR}/3270SemiCondensed-Regular.ttf $(DESTFOLDER)
 
 uninstall:
-	@$(RM) $(DESTFOLDER)/3270Narrow.ttf $(DESTFOLDER)/3270Medium.ttf $(DESTFOLDER)/3270SemiNarrow.ttf
+	@$(RM) $(DESTFOLDER)/3270Condensed-Regular.ttf $(DESTFOLDER)/3270-Regular.ttf $(DESTFOLDER)/3270SemiCondensed-Regular.ttf
 
 zip: derived
-	@zip -j ${BUILD_DIR}/3270_fonts_$(shell git rev-parse --short HEAD).zip ${BUILD_DIR}/3270Medium.* ${BUILD_DIR}/3270SemiNarrow.* ${BUILD_DIR}/3270Narrow.* LICENSE.txt fonts-3270.metainfo.xml
+	@zip -j ${BUILD_DIR}/3270_fonts_$(shell git rev-parse --short HEAD).zip ${BUILD_DIR}/3270-Regular.* ${BUILD_DIR}/3270SemiCondensed-Regular.* ${BUILD_DIR}/3270Condensed-Regular.* LICENSE.txt fonts-3270.metainfo.xml
 
-skimpytest: derived
-	fontlint ${BUILD_DIR}/3270Medium.otf
-	fontlint ${BUILD_DIR}/3270Medium.ttf
-	fontlint ${BUILD_DIR}/3270Medium.woff
-	fontlint ${BUILD_DIR}/3270SemiNarrow.ttf
-	fontlint ${BUILD_DIR}/3270Narrow.ttf
+fbchecks: derived
+	./fontbakery_checks.sh
 
-test: skimpytest
+skimpytest: derived fbchecks
```

Comment:
There's no convenient way to run fontlint but not fontbakery.  That's a problem as the latter is not packaged in any distribution (at least according to Repology), and running as many checks as possible is a good way to avoid regressions.
And the new versions happen to fail fontlint with `Bad PostScript fontname entry in the 'name' table`...

Reply:
pip install fontbakery will always give you a very up-to-date version of fontbakery. I make almost weekly releases of fontbakery on PyPI.

The distro packaging of tools is way too slow for fontbakery. We update it very frequently to fix all sorts of bugs and to keep the Google Fonts onboarding criteria up-to-date. Using an old version of fontbakery would actually be probably bad as it would give a wrong report of what's the expected level of quality of a font project for inclusion in the Google Fonts collection.

Reply:
the bad postscript name is an actual problem me and @rbanffy have been discussing. We will likely need to change the font name, since it is composed exclusively of numbers and that would likely crash a postscript program (the font name would end up being interpreted as an integer instead)

Reply:
by the way, it seems that even names simply starting with numbers are enough to confuse the postscript interpreter

Reply:
Yeah, you've explained that up-to-date thing to me in person. :)

In a good part of distributions (among them Debian) it's forbidden to reach outside network during the build; it's also forbidden to use non-packaged bits, for a long list of reasons.  Thus, I'll have to skip fontbakery checks.

It is also impossible for an old released version of a font to comply with new requirements that newly appeared in a future version of a checker.  The font, like any other package, must remain buildable, but that's done with contemporary versions of any checkers and toolchains.

Reply:
To fix the distro build issue, I'd suggest moving the fontbakery test to a separate build target, at least for now.

On the packaging of fontbakery, I see no reason why an outdated version would be worse than no version - the worst case scenario is that it wouldn't flag errors that are only tested in more current versions, but would still flag all previous ones. If the packaged version has erroneous tests in it, maintenance releases are still possible. I can help with backporting, if needed.